bungeecord/src/main/java/com/github/dominik48n/party/bungee/listener/SwitchServerListener.java

```diff
@@ -32,18 +33,26 @@
 public class SwitchServerListener implements Listener {
 
     private final @NotNull UserManager<ProxiedPlayer> userManager;
+    private final @NotNull Logger logger;
 
-    public SwitchServerListener(final @NotNull UserManager<ProxiedPlayer> userManager) {
+    public SwitchServerListener(final @NotNull UserManager<ProxiedPlayer> userManager, final @NotNull Logger logger) {
```

Comment:
Lombok Konstruktor vlt einsetzen

Reply:
The reason why this is not implemented can be found in the comment in the pull request.